hw/dv/tools/dvsim/sim.mk

```diff
@@ -99,6 +99,7 @@ ifneq (${sw_images},)
 			bazel_opts+=" --//hw/ip/otp_ctrl/data:img_seed=${seed}"; \
 			if [[ "${build_seed}" != "None" ]]; then \
 				bazel_opts+=" --//hw/ip/otp_ctrl/data:otp_seed=${build_seed}"; \
+				bazel_opts+=" --//hw/ip/otp_ctrl/data:lc_seed=${build_seed}"; \
```

Comment:
@timothytrippel can you PTAL whether this is right?

Reply:
yep this is right, but for this to have any effect, the build opt needs to be piped through the `otp_image` generation Bazel rule, as described below.

Reply:
I am a little confused. I thought the variable name that represents the seed value for the build stage is also `seed`, not `build_seed`.

Reply:
I think the connection from the build seed value from the Hjson to `Deploy::build_seed` is missing.

Reply:
I believe this `{build_seed}` value is made available in the templating environment by dvsim. I confirmed this was the case in #16052.

Reply:
I see. Let me read through later to understand it better,